submissions/opeyemiPeter/css/product.css

```diff
@@ -0,0 +1,309 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Montserrat;
+  font-size: 18px;
+  line-height: 22px;
+  overflow-x: hidden;
+}
+a {
+  text-decoration: none;
+}
+
+/********************
+        header
+**********************/
+header {
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: flex-end;
+  background: #1b1a1b;
+  align-items: center;
+  padding-top: 10px;
+  padding-bottom: 10px;
+}
+.logo {
+  margin-right:auto;
+  margin-left: 75px;
+}
+.logo a {
+  font-family: "", cursive;
```

Comment:
You already set the font family in the body tag. No need for it here. @opeyemipeter1759

Reply:
@ToluAkin i wanted a different fontfamily for the logo from that of the family